docs/source/contributing.rst

```diff
@@ -82,15 +82,21 @@ Installing a Development Version
       pip install -e .  # Installs SCICO from the current directory in editable mode
 
 
-9. The SCICO project uses the `Black <https://black.readthedocs.io/en/stable/>`_
+9. The SCICO project uses the `black <https://black.readthedocs.io/en/stable/>`_
    and `isort <https://pypi.org/project/isort/>`_ code formatting utilities.
-   You should set up a `pre-commit hook <https://pre-commit.com>`_ to ensure
-   any modified code passes format check before it is committed to the development repo:
+   It is important to set up a `pre-commit hook <https://pre-commit.com>`_ to
+   ensure that any modified code passes format check before it is committed to
+   the development repo:
 
    ::
 
       pre-commit install  # Sets up git pre-commit hooks
 
+   It is also recommended to `pin the conda package version
```

Comment:
The current requirement is `black>=21.6b0,<22` is that incorrect? I.e., is a version in that range not compatible with the action run on github?

If not, I don't think this problem should be affecting anyone with a normal SCICO install, because black should be `pip`-installed and conform to the version number in `dev_requirements.txt`.

Reply:
The problem is that subsequent updates (e.g. `conda update --all`) have a tendency to install a version of black outside of this specified range.